### DISCUSSION TITLE
List the MediaElement supported codecs?

### DISCUSSION THREAD

**Discussion by unliftedq** [CommunityToolkit/Maui#2114] [General]
I don't find any document for it, is it possible to have a document to list everything there, for example

|   Platform   |  Type   |  Codec      |
| ------------- | ------------- | ------------- |
| Windows | video | mp4 |
| Windows | audio | mp3 |
| Android  | video | mp4 |
| iOS | audio | mp3 |
| ... |  ... | ... |

**Comment by CliffAgius** [comment]
This is listed per platform in the docs:

https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/mediaelement?tabs=windows#supported-formats
